concourse/pipelines/create-deployer.yml

```diff
@@ -261,12 +272,13 @@ jobs:
 
     - task: terraform-apply
       config:
-        image: docker:///governmentpaas/terraform
+        image: docker:///governmentpaas/terraform#new-tf-build
```

Comment:
https://github.com/alphagov/paas-docker-cloudfoundry-tools/pull/42 was merged and the new image is already available on docker hub so you can remove this now.